tools/devtools/src/devtools/tabs/Document.tsx

```diff
@@ -60,7 +60,18 @@ export function Document({ style }) {
 
   return (
     <div className="yorkie-root content-wrap" style={{ ...style }}>
-      <div className="title">{currentDocKey || 'Document'}</div>
+      <div className="devtools-tab-toolbar">
+        <span className="title">{currentDocKey || 'Document'}</span>
+        <button
+          className="toggle-tab-btn"
+          onClick={() => {
+            setHidePresenceTab((v: boolean) => !v);
+          }}
+        >
+          {hidePresenceTab ? '⏴' : '⏵'}
```

Comment:
The icons appear as squares in my browser. Could you check if `◂` and `▸` are the intended icons?

Reply:
Oh, it seems like there is a compatibility issue with the icon because I intended the icon to look different. (as below image)

I think the easiest way to resolve this issue is to replace it with an SVG icon.

However, I'm not sure where to get those icons from due to copyright issues. Since there are already SVG icons in the repository, I was curious if we have a specific source or guidelines for using SVG icons in our project.
![Screenshot 2024-07-23 at 1 14 40 PM](https://github.com/user-attachments/assets/ebfe32a2-1944-4fda-bd1a-4d874c8ce519)

Reply:
The icons used in devtools were custom-made. https://www.figma.com/design/IBIiGml3fbfxgkSr14fSgy/Yorkie-(Sharing)?node-id=8090-216561&t=NrV28SDSYHMixFXy-0
For the Yorkie homepage design, we used free icons from https://www.untitledui.com/, which are also attached in the Figma file. Simple icons can be created directly, or we can use the UntitledUI icons.